contrib/cf-remote/cf_remote/remote.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python3
 import sys
+import os
 from os.path import basename
 from collections import OrderedDict
+import json
```

Comment:
I don't see neither `os` nor `json` in the added lines below. I guess some remnants of placing the code into a different file eventually?